h2/src/main/org/h2/mvstore/tx/TransactionStore.java

```diff
@@ -144,6 +144,12 @@ public void init() {
         if (!init) {
             for (String mapName : store.getMapNames()) {
                 if (mapName.startsWith(UNDO_LOG_NAME_PREFIX)) {
+                    if (mapName.equals(UNDO_LOG_NAME_PREFIX)) {
```

Comment:
a comment here would be very useful to future maintainers when we forget all about the format changes in this version :-)